packages/pyright-scip/snapshots/input/graph_1278/mwe.py

```diff
@@ -0,0 +1,21 @@
+from typing import TypeVar, Generic, Callable, Iterator, ParamSpec
```

Comment:
This test cases seems a bit large, but I haven't been able to minimize it further.